.github/workflows/version.yml

```diff
@@ -0,0 +1,37 @@
+name: "Bump Version"
```

Comment:
See my comment on your other PR about the issues this will cause:
https://github.com/zer0-os/event-listener/pull/25#discussion_r892907616

Reply:
This wont happen here because i configured circleci to only run on tags